platform/linux-generic/pktio/dpdk.c

```diff
@@ -1209,7 +1212,10 @@ static int dpdk_vdev_promisc_mode_get(uint16_t port_id)
 
 	memset(&dev_info, 0, sizeof(struct rte_eth_dev_info));
 
-	rte_eth_dev_info_get(port_id, &dev_info);
+	if (rte_eth_dev_info_get(port_id, &dev_info) != 0) {
+		_ODP_ERR("rte_eth_dev_info_get() failed for port %u\n", port_id);
```

Comment:
All new error messages could also include the function return value to ease possible debugging.